src/overlays/actors/ovl_En_Ru1/z_en_ru1.c

```diff
@@ -381,25 +381,25 @@ void func_80AEB220(EnRu1* this, PlayState* play) {
 }
 #endif
 
-void func_80AEB264(EnRu1* this, AnimationHeader* animation, u8 arg2, f32 morphFrames, s32 arg4) {
+void EnRu1_AnimationChange(EnRu1* this, AnimationHeader* animation, AnimationMode mode, f32 morphFrames, bool playReversed) {
```

Comment:
```suggestion
void EnRu1_AnimationChange(EnRu1* this, AnimationHeader* animation, AnimationMode mode, f32 morphFrames, s32 playReversed) {
```
we don't use bool as a type in the repo because the og devs used all kinds of sizes for effectively booleans